symfony/plugins/orangehrmMaintenancePlugin/test/Dao/MaintenanceDaoTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace OrangeHRM\Maintenance\test\Dao;
```

Comment:
Please add the copyright header
```suggestion
namespace OrangeHRM\Tests\Maintenance\Dao;
```